src/Components/Movies.js

```diff
@@ -0,0 +1,85 @@
+import axios from "axios";
+import React, { Component } from "react";
+import "./Movies.css";
+
+export class Movies extends Component {
+  constructor() {
+    super();
+    this.state = {
+      movies: [],
+      failedApiCall: false,
+      selectedTitle: "",
+      selectedRelease: "",
+      selectedDescription: "",
+    };
+  }
+
+  componentDidMount() {
+    this.getMovies();
+  }
+
+  getMovies = async () => {
+    try {
+      const { data } = await axios.get(`https://ghibliapi.herokuapp.com/films`);
+      this.setState({
+        movies: data,
+        failedApiCall: false,
+      });
+    } catch {
+      this.setState({
+        failedApiCall: true,
+      });
+    }
+  };
+
+  chooseMovie = (e) => {
+    const { value } = e.target;
+    const { movies } = this.state;
+
+    if (value) {
+      let movie = movies.filter((movie) => movie.id === value);
```

Comment:
You basically invented JavaScript's `.find` array method here. If you don't know it, check it out; it will replace this sweet `.filter` call with an even sweeter syntax.